.github/workflows/test_frontend_e2e_live.yml

```diff
@@ -1,7 +1,7 @@
 # TODO: 1. run local: (i) start backend (a) local backend in GH action, or (b) use dev's backend URL (won't always work), (ii) set to on:pr&push
 # TODO: 2. after '1': expose params to workflow_dispatch: so can choose which env we want to test, in case we don't want to run all 3 at once
 
-name: Tests and QC - Frontend
+name: End to end tests - Frontend deployments
```

Comment:
## Playwright action updates
- [ ] Playwright E2E action: check that still runs

I need to makes sure it still works, because the setup in `package.json` has changed, which also meant changes in the `makefile` and the action.
How I test this: (i) Create action w/ the same name on `develop` (empty or copy paste). Then I should be able to run action on this branch.